### PR TITLE
added sbc upload file rake task

### PR DESCRIPTION
**Pull request by Rohithone**
https://www.pivotaltracker.com/n/projects/2481183/stories/184082174

I have added rake tasks and spec files, please review and here is the rake command to run the sbc rake

`RAILS_ENV=development bundle exec rake sbc:upload -- -csv_path 'db/seedfiles/plan_xmls/me/sbcs/2023/2023_sbcs.csv' -dir_path 'db/seedfiles/plan_xmls/me/sbcs/2023/'
`

### CODE REVIEW

File: lib/sbc/sbc_processor.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          puts "FILE NOT FOUND #{product.title} #{product.hios_id} #{file_name} "
+          next
+        end
+
+        uri = if Rails.env.test?
+                'urn:openhbx:terms:v1:file_storage:s3:bucket:mhc-enroll-sbc-test#11111111-1111-1111-1111-111111111111'
+              else
+                Aws::S3Storage.save(pdf_path(file_name), S3_BUCKET)
+              end
+        product.sbc_document = Document.new(title: file_name, subject: 'SBC', format: 'application/pdf', identifier: uri)
```
Comment (nisanthyaganti9) [inline lib/sbc/sbc_processor.rb:51]: update `::Document` with `::Documents::Document.`

File: lib/sbc/sbc_processor.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      products.each do |product|
+        file_name = row[1].strip
+
+        if pdf_path(file_name).nil?
+          puts "FILE NOT FOUND #{product.title} #{product.hios_id} #{file_name} "
+          next
+        end
+
+        uri = if Rails.env.test?
+                'urn:openhbx:terms:v1:file_storage:s3:bucket:mhc-enroll-sbc-test#11111111-1111-1111-1111-111111111111'
```
Comment (nisanthyaganti9) [inline lib/sbc/sbc_processor.rb:47]: remove `mhc` and get this from resource registry

File: lib/tasks/upload_sbc.rake [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+require 'csv'
+require Rails.root.join('lib', 'sbc', 'sbc_processor')
+
+ENV_LIST = %w[qa prod preprod cte cpr uat].freeze
+
+namespace :sbc do
+  desc 'Uploading Sbc Document'
+  task :upload, [:csv_path, :dir_path] do |_task, args|
+    # task :upload, [:csv_path, :dir_path] => :environment do |task, args|
```
Comment (nisanthyaganti9) [inline lib/tasks/upload_sbc.rake:11]: remove this comment

File: spec/models/aws/s3_storage_spec.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Aws::S3Storage do
+  # allow(:storage_double) {double}
+  # allow(Aws::S3Strorage).to receive(:new).and_respond_with(storage_double)
```
Comment (nisanthyaganti9) [inline spec/models/aws/s3_storage_spec.rb:7]: remove these comments

**Review by nisanthyaganti9 — CHANGES_REQUESTED** [review]

File: spec/models/aws/s3_storage_spec.rb (hunk 4 above)
Comment (Rohithone) [reply spec/models/aws/s3_storage_spec.rb]: have removed comments and pushed changes

**Review by nisanthyaganti9 — APPROVED** [review]

**Review by rahulgudim — APPROVED** [review]

**Pull request merged**